src/lwc/geTemplateBuilder/geTemplateBuilder.js

```diff
@@ -2,6 +2,7 @@ import { LightningElement, track, api, wire } from 'lwc';
 import { NavigationMixin } from 'lightning/navigation';
 import storeFormTemplate from '@salesforce/apex/GE_GiftEntryController.storeFormTemplate';
 import retrieveFormTemplateById from '@salesforce/apex/GE_GiftEntryController.retrieveFormTemplateById';
+import checkForElevateCustomer  from '@salesforce/apex/GE_GiftEntryController.isElevateCustomer';
```

Comment:
I think we can remove this since we're now getting `isElevateCustomer` from `geTemplateBuilderService`

Reply:
Removed!